printers/prusa-mk3s/tmc2130.cfg

```diff
@@ -0,0 +1,70 @@
+# WARNING. DO NOT EDIT THIS FILE.
+# To override settings from this file, you can copy and paste the relevant
+# sections into your printer.cfg and change it there.
+
+[tmc2130 stepper_x]
+cs_pin: x_cs_pin
+interpolate: False
+diag1_pin: !x_diag_pin
+driver_IHOLDDELAY: 8
+driver_TPOWERDOWN: 0
+driver_TBL: 2
+driver_TOFF: 3
+driver_HEND: 1
+driver_HSTRT: 5
+driver_PWM_FREQ: 2
+driver_PWM_GRAD: 2
+driver_PWM_AMPL: 230
+driver_PWM_AUTOSCALE: True
+driver_SGT: 5
+
+
+[tmc2130 stepper_y]
+cs_pin: y_cs_pin
+interpolate: False
+diag1_pin: !y_diag_pin
+driver_IHOLDDELAY: 8
+driver_TPOWERDOWN: 0
+driver_TBL: 2
+driver_TOFF: 3
+driver_HEND: 1
+driver_HSTRT: 5
+driver_PWM_FREQ: 2
+driver_PWM_GRAD: 2
+driver_PWM_AMPL: 235
+driver_PWM_AUTOSCALE: True
+driver_SGT: 5
+
+
+[tmc2130 stepper_z]
+cs_pin: z_cs_pin
+interpolate: False
+diag1_pin: !z_diag_pin
+driver_IHOLDDELAY: 8
+driver_TPOWERDOWN: 0
+driver_TBL: 2
+driver_TOFF: 3
+driver_HEND: 1
+driver_HSTRT: 5
+driver_PWM_FREQ: 2
+driver_PWM_GRAD: 4
+driver_PWM_AMPL: 200
+driver_PWM_AUTOSCALE: True
+driver_SGT: 4
+
+[tmc2130 extruder]
+cs_pin: e_cs_pin
+interpolate: True
+sense_resistor: 0.220
+diag1_pin: !e_diag_pin
+driver_IHOLDDELAY: 8
+driver_TPOWERDOWN: 0
+driver_TBL: 2
+driver_TOFF: 3
+driver_HEND: 1
+driver_HSTRT: 5
+driver_PWM_FREQ: 2
+driver_PWM_GRAD: 4
+driver_PWM_AMPL: 240
+driver_PWM_AUTOSCALE: True
+driver_SGT: 3
```

Comment:
driver_* settings still need to go into the appropriate steppers/* file, so steppers/ldo/42sth40-1684/2130/24v-0.4a-x.cfg for example, since they're specific to the driver/stepper combination.

Reply:
Done